db/migrate/20161017135917_create_organizations.rb

```diff
@@ -0,0 +1,16 @@
+class CreateOrganizations < ActiveRecord::Migration
+  def change
+    create_table :organizations do |t|
+      t.string :name
+      t.string :display_name
+      t.string :slug, default: ""
+      t.string :primary_language, null: false
+      t.timestamp :listed_at, default: nil
+      t.integer  :activated_state, default: 0, null: false
+
+      t.timestamps null: false
+    end
+
+    add_index :organizations, :listed_at, where: "(listed_at IS NOT NULL)"
```

Comment:
with that index in place you should be able to run `Organisation.public.explain` & `Organisation.private.explain` and see if they use the indexes you've got. If you're in a testing env then the query planner may want to use a table scan as it'll be cheaper. 

You can force pg to not table scan for the session with `SET enable_seqscan = OFF;`